pkg/ddevapp/ddevapp.go

```diff
@@ -1639,7 +1639,7 @@ func (app *DdevApp) ExecWithTty(opts *ExecOpts) error {
 
 	state, err := dockerutil.GetContainerStateByName(fmt.Sprintf("ddev-%s-%s", app.Name, opts.Service))
 	if err != nil || state != "running" {
-		return fmt.Errorf("service %s is not current running in project %s (state=%s)", opts.Service, app.Name, state)
+		return fmt.Errorf("service %s is currently not running in project %s (state=%s)", opts.Service, app.Name, state)
```

Comment:
I think this might be a little simpler.

```suggestion
		return fmt.Errorf("service %s is not running in project %s (state=%s)", opts.Service, app.Name, state)
```

Reply:
Ahhhh I thought currently should kept in. But removing it is short and even more clear. Good point I'll change that. But one question i've added in the initial comment. Should "service" be upper case since it is after a colon? Stylistically it is recommended but i suppose out of the scope of this issue. Cuz i think it would require to update a few more strings. Maybe something for a followup?

Reply:
I think it's probably fine like this. Thanks!